src/Dotnet.Script/Program.cs

```diff
@@ -81,16 +81,16 @@ private static void RunScript(string file, string config, bool debugMode, IEnume
             }
 
             var directory = Path.IsPathRooted(file) ? Path.GetDirectoryName(file) : Path.GetDirectoryName(Path.Combine(Directory.GetCurrentDirectory(), file));
-            var sourceText = SourceText.From(new FileStream(file, FileMode.Open));
-            var context = new ScriptContext(sourceText, directory, config, args, file);
+            var sourceText = SourceText.From(new FileStream(file, FileMode.Open),Encoding.UTF8);
```

Comment:
you should not hardcode the encoding. `SourceText.From(stream)` will internally try to detect encoding using BOM and if it's not found, it will fallback to UTF8 automatically